pipeline-templates/job/deploy.yml

```diff
@@ -90,6 +90,12 @@ jobs:
             ResourceName: $(apiAppServiceName)
             Tenant: $(Tenant)
             IsMultiRepoCheckout: true
+        - template: azure-pipelines-templates/deploy/step/app-role-assignments.yml@das-platform-building-blocks
+          parameters:
+            ServiceConnection: ${{ parameters.AppRoleAssignmentsServiceConnection }}
+            ResourceName: $(externalApiAppServiceName)
```

Comment:
ResourceName needs changing to ExternalApiAppServiceName

Reply:
@debrr07 Ok corrected that and the ApiAppServiceName also for consistency but the "apiAppServiceName" having been incorrect for some time suggests these are not actually case sensitive